src/features/prompts/server/prompt-router.ts

```diff
@@ -27,18 +27,44 @@ export const promptRouter = createTRPCRouter({
         projectId: input.projectId,
         scope: "prompts:read",
       });
-      return ctx.prisma.prompt.findMany({
+      const prompts = await ctx.prisma.$queryRaw<Array<Prompt>>`
+        SELECT id, name, version, created_at AS "createdAt", is_active AS "isActive"
+        FROM prompts
+        WHERE (name, version) IN (
+          SELECT name, MAX(version)
+          FROM prompts
+          WHERE "project_id" = ${input.projectId}
+          GROUP BY name
+        )
+        AND "project_id" = ${input.projectId}`;
```

Comment:
WDYT about ordering this by name? This way the ordering of the prompts table remains the same even if we create new prompts on any of these. Currenlty, the prompt with the latest addition is added on top.